packages/engine/Source/Scene/I3SField.js

```diff
@@ -92,40 +100,50 @@ function getNumericTypeSize(type) {
   return 0;
 }
 
+function getValueTypeSize(type) {
+  if (type === "String") {
+    return 1;
+  }
+  return getNumericTypeSize(type);
+}
+
+async function load(field) {
+  const data = await field._dataProvider._loadBinary(field._resource);
+
+  // Check if we have a 404
+  const dataView = new DataView(data);
```

Comment:
If the request 404's the `loadBinary` function should throw an exception instead of returning the binary.

Then, you should be able to use a `try/catch` block here instead of attempting to parse the results directly.

Reply:
Done.